plexapi/media.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
-import xml
 from pathlib import Path
 from urllib.parse import quote_plus
+from xml.etree import ElementTree
```

Comment:
You need to update the exception below with the new import.

```py
        except ElementTree.ParseError: 
```

https://github.com/pushingkarmaorg/python-plexapi/blob/8894883cda07d2d72410f40fbde64e43937c69ed/plexapi/media.py#L1078